notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+commands:
```

Comment:
Question about the use case. Will this be added to every job? I think it is quite common for people to push stuff to github which will trigger circle jobs that fail, and could result in the notifications becoming noisy. Perhaps there is a JIRA ticket or something describing this new feature you can link so I can better understand? 😄

Reply:
The Jira ticket is [this one](https://switchcomm.atlassian.net/browse/VAE-935). This new command has two conditions:
- _It MUST be used only on production deploys_: Since we could not reach a solution in which the slack message is triggered after all the workflow jobs, sending a message with all those jobs that fail, we decide to use it only on production.
- _It MUST be used as a `post-step` from a job_: that's why it is a command and not a job.

Perhaps, this information should be written on line 12 as a comment for a better understanding. What do you think?

Reply:
A comment with the specifics would be great -- as a general rule, we tend to look at the source code for an understanding of how something will work / any caveats / etc.

I would also recommend including all or part of this in the command's `description` field, as that would be visible to any usage downstream and in the published OSS docs (eg. [this page](https://circleci.com/developer/orbs/orb/talkiq/notifier)).

Reply:
I added these two bullet points on orb.yaml to clarify. On the description field, I added: "Note that if a job uses parallelism or matrix, it will send as many messages as jobs executed."